contrib/systemd/sys-fs-bpf.mount

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Cilium BPF mounts
+Documentation=http://cilium.readthedocs.io/en/latest/
```

Comment:
Can you refer to docs.cilium.io instead? In case we ever move away from readthedocs.io